dev/cli.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+git_id=$(git rev-parse --short=7 HEAD)
+
+image=docker.montagu.dide.ic.ac.uk:5000/orderly-web-cli:fixcli
```

Comment:
Once https://github.com/vimc/orderly-web/pull/41 is merged can change hard-coded `fixcli` to `$git_id`